pkg/apis/scheduling/types.go

```diff
@@ -57,7 +57,20 @@ const (
 type PodGroupConditionType string
 
 const (
+	// PodGroupUnschedulableType is Unschedulable event type
 	PodGroupUnschedulableType PodGroupConditionType = "Unschedulable"
+
+	// PodGroupScheduled is scheduled event type
+	PodGroupScheduled PodGroupConditionType = "Scheduled"
+)
+
+type PodGroupSchedulingStatus string
+
+const (
+	// PodGroupReady is that PodGroup has reached scheduling restriction
```

Comment:
scheduling condition or scheduling requirement?

Reply:
changed to `PodGroupConditionDetail` according to type of events `PodGroupConditionType`